src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -125,9 +126,11 @@ def _parse_addr(value):
         #   Group 1: [
         #   Group 2: 2001:db8:85a3::8a2e:370:7334
         addr = match.group(3) if match.group(3) else match.group(2)
-        if not is_valid_ip_address(addr):
+        try:
+            if ipaddress.ip_address(addr):
+                return addr
```

Comment:
Based on the [`ipaddress` library docs](https://docs.python.org/dev/library/ipaddress.html#ipaddress.ip_address), `ip_address` either returns a valid IP address object or triggers a `ValueError` exception, so `None`/`False` is not a possible return value. Am I missing something?

Reply:
Code should look like this:
```
try:
    ipaddress.ip_address(addr)
    return addr
except ValueError:
```

Reply:
@epuertat, @votdev yes, this has been taken care. The latest `make check` error is in `doctest`, do you think it is relevant ?

Reply:
> @epuertat, @votdev yes, this has been taken care. The latest `make check` error is in `doctest`, do you think it is relevant ?

Yes it is relevant. The tests MUST succeed, otherwise this PR can't be accepted.